docs/docs/30-administration/10-configuration/11-backends/20-kubernetes.md

```diff
@@ -6,6 +6,20 @@ toc_max_heading_level: 2
 
 The Kubernetes backend executes steps inside standalone Pods. A temporary PVC is created for the lifetime of the pipeline to transfer files between steps.
 
+## Metadata labels
+
+Woodpecker adds some labels to the pods to provide additional context to the workflow. These labels can be used for various purposes, e.g. for simple debugging or as selectors for network policies.
+
+The following metadata labels are supported:
+
+- `woodpecker-ci.org/forge-id`
+- `woodpecker-ci.org/repo-forge-id`
+- `woodpecker-ci.org/repo-id`
+- `woodpecker-ci.org/repo-name`
```

Comment:
Just to confirm, are there actually use-cases for `repo-forge-id`, `repo-id` and `repo-name` if you could use `repo-full-name`? I could image users want to filter for a specific repo, an org and maybe a forge. Having one unique selector for each like id or name should be sufficient, shouldn't it?

Reply:
Make it human readable. There was a huge discussion around how to debug containers as its pretty hard to find out which container/pod handles a step or workflow.

Reply:
Will check on the discussion and open a PR before the next release if necessary.